cnxpublishing/views/admin.py

```diff
@@ -271,3 +534,150 @@ def admin_edit_site_message_POST(request):
     args = admin_edit_site_message(request)
     args['response'] = "Message successfully Updated"
     return args
+
```

Comment:
This is a function from the site messages PR, is this something that slipped through.